src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1680,6 +1711,55 @@ void GrpcLb::CreateOrUpdateChildPolicyLocked() {
   child_policy_->UpdateLocked(std::move(update_args));
 }
 
+//
+// subchannel caching
+//
+
+void GrpcLb::CacheDeletedSubchannel(
```

Comment:
Looks like `StartSubchannelCacheTimer` and this method need to be synchronized the same way as `OnSubchannelCacheTimerLocked`, in order to safely access `cached_subchannels_`. So let's suffix these methods with `Locked`?

Reply:
Done.